cmd/plugins/juju-metadata/validatetoolsmetadata.go

```diff
@@ -94,28 +95,27 @@ Examples:
 A key use case is to validate newly generated metadata prior to deployment to
 production. In this case, the metadata is placed in a local directory, a cloud
 provider type is specified (ec2, openstack etc), and the validation is performed
-for each supported series, version, and arcgitecture.
+for each supported series, version, and architecture.
 
 Example bash snippet:
 
 #!/bin/bash
 
 juju metadata validate-agents -p ec2 -r us-east-1 -s precise --juju-version 1.12.0 -d <some directory>
 RETVAL=$?
-[ $RETVAL -eq 0 ] && echo Success
-[ $RETVAL -ne 0 ] && echo Failure
+[ $RETVAL -eq 0 ] && echo Success || echo "Failure"
 `
 
-func (c *validateToolsMetadataCommand) Info() *cmd.Info {
+func (c *validateAgentsMetadataCommand) Info() *cmd.Info {
 	return jujucmd.Info(&cmd.Info{
 		Name:    "validate-agents",
-		Purpose: "validate agent metadata and ensure agent binary tarball(s) exist for Juju version(s)",
-		Doc:     validateToolsMetadataDoc,
+		Purpose: "validate agent metadata and ensure agent binary .tgz tarball(s) exist for Juju version(s)",
```

Comment:
```suggestion
		Purpose: "validate agent metadata and check that compressed tar archives (.tgz) with the Juju agent binaries are available",
```

I think there was a bit of duplication there:
.tgz = (compressed) tar archive
tarball = tar archive